components/function-controller/design/glossary_proposal.md

```diff
@@ -0,0 +1,105 @@
+# Serverless naming convention
+
+## Problem overview
+
+Currently, Serverless in Kyma consists of two projects:
+
+- [function-controller](https://github.com/kyma-project/kyma/tree/main/components/function-controller) -
+  responsible for running a Function on a k8s cluster
```

Comment:
```suggestion
  responsible for running a Function on a Kubernetes cluster
```
I'd use Kubernetes instead of k8s, [as our guidelines suggest](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalization)